index.html

```diff
@@ -123,101 +124,120 @@ <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                        <li><a href="#" class="card__tags-links">Gitlab</a></li>
+                        <li><a href="#" class="card__tags-links">Terminal</a></li>
+                    </ul>
+                </div>
+            </div>
         </div>
-        <div class="contact__form">
-            <form action="https://formspree.io/f/mrgdrewl"
-            method="POST">
-                <div class="form-control">
-                    <label>
-                        <input name="full_name" type="text" placeholder="Full name" required maxlength="30">
-                    </label>
+    </section>
+
+    <!-- Contact -->
+    <section class="contact">
+        <div class="container">
+            <div class="contact__content">
+                <div class="contact__text">
+                    <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
                 </div>
-                <div class="form-control">
-                    <label>
-                        <input name="email" type="email" placeholder="Email Address" required>
-                    </label>
+                <div class="contact__form">
+                    <form action="https://formspree.io/f/mrgdrewl"
+                    method="POST">
+                        <div class="form-control">
+                                <input name="full_name" type="text" placeholder="Full name" required maxlength="30">
+                        </div>
+                        <div class="form-control">
+                                <input name="email" type="email" placeholder="Email Address" required>
+                        </div>
+                        <div class="form-control">
+                                <textarea name="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
+                        </div>
+                        <button class="btn btn__submit" type="submit">Get in touch</button>
+                    </form>
                 </div>
-                <div class="form-control">
-                    <label>
-                        <textarea name="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
-                    </label>
+                <div class="contact__form-desktop">
+                    <form action="https://formspree.io/f/mrgdrewl"
+                    method="POST">
+                        <div class="contact__name">
+                            <div class="form-control">
+                                <input name="first_name" type="text" placeholder="First Name" required maxlength="30">
+                        </div>
+                        <div class="form-control">
+                                <input name="last_name" type="text" placeholder="Last Name" required maxlength="30">
+                        </div>
```

Comment:
- [ ] I know we have been asking you to stick to the Figma design all time around, but here comes an exception. 😎️
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) we are supposed to ignore the Figma design ordering us to create `First name` and `Last name` and replace it with a `Full name` so we can use the same form on multiple displays. 🤝️
Kindly remove the two inputs and replace them with a `Full name` input that takes the entire space taken by both of them.

**Project Requirements**

![Screenshot from 2023-01-20 09-07-50](https://user-images.githubusercontent.com/17243325/213631638-8965fe6a-ac79-4eea-bb2f-6dd0ab8a872c.png)